store/views.py

```diff
@@ -12,58 +15,71 @@ def index(request):
 
 def bookDetailView(request, bid):
     template_name = 'store/book_detail.html'
+    book1=Book.objects.get(pk=bid)
+    num_available=BookCopy.objects.filter(status__exact=True,book__exact=book1).count()
+    user_rating=0.0
+    if(request.user.is_authenticated):
+        value= BookRating.objects.filter(book__exact=book1,user__exact=request.user)
+        user_rating=0
+        if(value.count()>0):
+            user_rating=BookRating.objects.filter(book__exact=book1,user__exact=request.user).get().rating
```

Comment:
This is not a good way to call ORM queries. This will run two queries on database. You could have directly done the following:
```python
user_rating = value.get().rating
```